cmd/mapkubeapis/map.go

```diff
@@ -52,8 +52,11 @@ func newMapCmd(out io.Writer, args []string) *cobra.Command {
 		Long:         "Map release deprecated or removed Kubernetes APIs in-place",
 		SilenceUsage: true,
 		Args: func(cmd *cobra.Command, args []string) error {
-			if len(args) != 1 {
-				return errors.New("name of release to be mapped needs to be passed")
+			if len(args) == 0 {
+				cmd.Help()
+				os.Exit(0)
```

Comment:
Sorry missed this but should this be `os.Exit(1)` ?

Reply:
Yeah, no problem. I wasn't really sure what the standard was since looking at different tools, it was inconsistent on whether printing the help on no args was an error state or not